### PR TITLE
DOCSP-30987: use FCV instead of fCV

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
# Pull Request Process

Please read the
[Style Guide Review Process](https://wiki.corp.mongodb.com/display/DE/Technical+Writer+Style+Guide+Review+Process)
wiki page.

Contributors should take the following actions:

- Request a PR review in #docs-style-guide Slack channel.
- Add the Style Guide team reps as reviewers.
- If any reviewer requests changes, address them and request another review.
- When you receive PR review approvals from all the Style Guide team reps,
  they will merge the PR and notify you of the merge in the Slack channel.

# Pull Request Description

replacing ``fCV`` with ``FCV``

JIRA URL: <https://jira.mongodb.org/browse/DOCSP-30987>

Staging URL: 

- https://deploy-preview-178--mongodb-docs-meta.netlify.app/style-guide/terminology/alphabetical-terms/#std-term-FCV
- https://deploy-preview-178--mongodb-docs-meta.netlify.app/style-guide/style/capitalization/#capitalize-most-abbreviations

### CODE REVIEW

File: source/style-guide/includes/terms/f.rst [hunk 1]
```diff
@@ -36,10 +36,10 @@ F
    fatal
      :icon-fa4:`times-circle` Use *serious* instead.
 
-   fCV
+   FCV
      A MongoDB term that describes the "feature compatibility version"
```
Comment (corryroot) [inline source/style-guide/includes/terms/f.rst:40]:
```suggestion
    MongoDB abbreviation for the "feature compatibility version"
```

**Review by corryroot — APPROVED** [review]
LGTM with a suggestion for the preexisting text. Thanks @jwilliams-mongo!

**Review by stephmarie17 — APPROVED** [review]
LGTM with the suggestion from @corryroot ✅

**Review by lindseymoore — APPROVED** [review]
Thanks LGTM!

**Pull request merged**